code/stable/gooltest/julia/HelloWorld/HelloWorld.jl

```diff
@@ -54,7 +54,7 @@ mySlicedList4 = myOtherList[4:2 - 1]
 
 mySlicedList5 = myOtherList[4:-1:2 - 1]
 
-mySlicedList6 = myOtherList[start:-1:2 - 1]
+mySlicedList6 = myOtherList[begin:-1:2 - 1]
```

Comment:
Any reason for this? Just curious!

Reply:
I think I accidentally wrote `start` instead of `begin` a while ago, and just didn't catch it till now 😅

Reply:
Is `begin` a keyword in Julia?

Reply:
yeah, when dealing with lists, `begin` and `end` refer to the first and last indices of the list!